### PR TITLE
Adding backup and license files for graphics + Resolve Styling comments   

### PR DESCRIPTION
**Pull request by martinstacey**
- Adding backup for all the graphics created and their licenses.
- Adding punctuation to bullet points.
- Rephrasing sentences in section: Why should I use generative design.
- Changing the image of gears to illustrate different values.
- Change wording of some sentences in _what is generative design_
- Changing wording of some sentences in _anatomy of each stage_
- Making lego girl image a little bit older by adding glasses
- Changing wording of some sentences in _mars example_

### CODE REVIEW

**Comment by radumg** [comment]
PR name and description need to be updated to reflect scope - I'll be able to review after that.

**Comment by martinstacey** [comment]
Pr name and description was updated to reflect scope

File: 01-introduction/01-01_computational-design.md [hunk 1]
```diff
@@ -10,5 +10,5 @@
 
 Put simply, computers are very good at **performing calculations** and **executing pre-defined steps.**
 
-When approaching a design computationally, the procedure used to create the design is the output of a designer’s work, not the resulting object itself. As such, the process of iterating project geometry and data are offloaded to a machine, saving time, money, and effort, and letting the designer focus on the creativity of the design process.
+When approaching a design computationally, the designer would focus on developing the procedure that would create a design, and not the design in itself. The process of iterating through options and data are offloaded to a machine. Saving time, money, and effort, and letting the designer focus on the creativity of the design process.
```
Comment (radumg) [inline 01-introduction/01-01_computational-design.md]: use `design itself` - `in` is not needed.

Comment (radumg) [inline 01-introduction/01-01_computational-design.md]: can't start a sentence with `Saving` and not connect it to previous one somehow.
Use `This saves time...` ... `and lets the designer focus...`

Comment (radumg) [inline 01-introduction/01-01_computational-design.md]:
```suggestion
When approaching a design computationally, the designer would focus on developing the procedure that would create a design, and not the design itself. The process of iterating through options and data are offloaded to a machine. This saves time, money, effort, and lets the designer focus on the creativity of the design process.
```

File: 01-introduction/01-02_generative-design/01-02-02_why-should-i-use-generative-design.md [hunk 2]
```diff
@@ -26,15 +26,14 @@ Generative design can help a designer find better designs for his or her project
...
 <img src="../../assets/intro/whyusegen3.gif" style="width:200px;"/>
 
 >Design options generated - Bionic Partition for Airbus - The Living
 
 ## Collaborative
 
-An important point to note is that the aim of a generative approach is to **augment human capability** with computer power, not to replace designers.
-
-To help illustrate this, remember that a good generative design process that uses several metrics to analyze designs in a study will rarely generate a single output. It  will almost always generate a wide range of outputs from which the designer can choose. That is, it doesn’t choose for the user, instead it gives you a list of options to choose from, with the added benefit that they have already been objectively assessed and ranked.
+The aim of a generative approach is to **augment human capability** with computer power, not to replace designers.
```
Comment (radumg) [inline 01-introduction/01-02_generative-design/01-02-02_why-should-i-use-generative-design.md]:
```suggestion
The aim of a generative approach is to **augment human capability** with computation power, not to replace designers.
```

File: 01-introduction/01-02_generative-design/01-02-03_what-goes-into-a-generative-design-process/01-02-03-01_anatomy-of-each-stage.md [hunk 3]
```diff
@@ -29,14 +29,14 @@ Things that are generated in the run step are the final results or outputs of ea
 
 ## Overall process
 
-We can map these stages & steps together in a single diagram to visualize the order they happen in and the dependencies between them.
+We can map these stages and steps together in a single diagram, this way we are able to visualize the order of each stage and their dependencies.
```
Comment (radumg) [inline 01-introduction/01-02_generative-design/01-02-03_what-goes-into-a-generative-design-process/01-02-03-01_anatomy-of-each-stage.md]: doesn't read nice as a written sentence, feels like `speech to text`.
```suggestion
We can map these stages and steps together in a single diagram, allowing us to visualize the order of each stage and their dependencies.
```

File: 01-introduction/01-02_generative-design/01-02-04_examples-of-generative-design/01-02-04-01_mars-innovation-district-of-toronto.md [hunk 4]
```diff
@@ -31,12 +29,12 @@ To begin with, information was collected from employees and managers about work
...
+After the algorithms were formulated, the computer used them to evaluate each of the designs generated in the previous stage against the defined goals.
 
 ### Explore
 
 <img src="../../../assets/intro/mars3.gif" style="width:200px;"/>
 
 > Design Options - Mars Innovation District - The Living
 
-Once designs have been evaluated, the designers then explore what is often called the _`solution space`_ : the generated designs together with their evaluation results. Taking into account each defined goal, they identified the design that best achieved the defined goals.
+After the designs were evaluated, the designers explored the _`solution space`_ : (the generated designs together with their evaluation results). Taking into account each defined goal, they identified the design that best achieved the goals overall.
```
Comment (radumg) [inline 01-introduction/01-02_generative-design/01-02-04_examples-of-generative-design/01-02-04-01_mars-innovation-district-of-toronto.md]: colon or bracket, not both
```suggestion
After the designs were evaluated, the designers explored the _`solution space`_ (the generated designs together with their evaluation results). Taking into account each defined goal, they identified the design that best achieved the goals overall.
```

File: assets/Illustrator Files/RefineryGraphics261119/RefineryGraphics261119 Report.txt [hunk 5]
```diff
@@ -0,0 +1,168 @@
+Package Report
```
Comment (radumg) [inline assets/Illustrator Files/RefineryGraphics261119/RefineryGraphics261119 Report.txt]: this file is not useful and contains references to your individual computer and `Dropbox` folder. Please remove this from PR.

File: 01-introduction/01-01_computational-design.md (hunk 1 above)
Comment (radumg) [reply 01-introduction/01-01_computational-design.md]: left a suggestion you can use below.

Comment (radumg) [reply 01-introduction/01-01_computational-design.md]: use suggestion below

**Comment by martinstacey** [comment]
Ready for review @radumg

**Comment by martinstacey** [comment]
@radumg All font files were deleted.

**Pull request merged**